Git Formats/Git Mailmap.sublime-syntax

```diff
@@ -12,14 +12,16 @@ scope: text.git.mailmap
 contexts:
   main:
     - include: Git Common.sublime-syntax#comments
-    - include: Git Common.sublime-syntax#email
-    - match: (<)(\S+?(?:(@)\S+)?)(>)
+    # email reference
+    - match: (<)(\S+?(?:(\@)\S+?(?:(\.)\S+?)?)?)(>)
+      scope: meta.reference.email.git
```

Comment:
Is this typical? I don't usually look for `scope:` if there are `captures:`.

Reply:
The match was copied from _Git Common.sublime-syntax_ and modified to make the `@` and `.` parts optional. The `scope` was already there. It has the same effect as adding a `0: meta.reference.email.git` to the list of captures.

I can't answer the question whether it is typical or not, but I've seen it in a couple of syntax definitions and got used to it myself in order to distinguish scopes which cover the whole match from those which match to single parts only. Always tried to avoid adding a `0: ...` capture.

I guess it is a question of taste in the end.

Reply:
See PackageDev's syntaxes for instance.

Reply:
> I can't answer the question whether it is typical or not, but I've seen it in a couple of syntax definitions and got used to it myself in order to distinguish scopes which cover the whole match from those which match to single parts only. Always tried to avoid adding a `0: ...` capture.

I checked the rest of the Packages project. It looks like you're right. There are 487 instances of `scope` and `captures` on the same match, versus 75 instances of `captures` with a `0` and another number. (There are also a bunch of `0`-`captures` that _do not need it,_ but that's a topic for another PR. 😉